internal/crypto/rewrap.go

```
@@ -0,0 +1,227 @@
+			return err
+		}
+		return err
+	}
+	// Sync the parent directory to persist the rename.
+	dir := filepath.Dir(path)
+	d, err := os.Open(dir)
+	if err != nil {
+		return err
+	}
+	if err = d.Sync(); err != nil {
+		d.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `d.Close` is not checked

(errcheck)